plugin-client/src/main/java/com/cloudera/nav/plugin/client/NavApiCient.java

```diff
@@ -71,18 +78,105 @@ public void registerModels(Collection<Object> models) {
    * @return a collection of available sources
    */
   public Collection<Source> getAllSources() {
+    System.out.println("here");
```

Comment:
Take out this debug print statement. For debugging it'll be a lot easier to use breakpoints in your IDE instead of these print statements